test/test.js

```diff
@@ -630,6 +630,21 @@ var util = require('util')
         // act
         cors()(req, res, next);
       });
+
+      it('should allow origin if array include *', function (done) {
+        var options = {
+          origin: ['*']
```

Comment:
I don't think that `origin: ['*']` should mean _reflect back the requesting origin_ when `origin: '*'` means _send a header of Access-Control-Allow-Origin: *_. I have some more thoughts [here](https://github.com/expressjs/cors/issues/185#issuecomment-673030304).